### PR TITLE
Pi updates

### PR DESCRIPTION
**Pull request by RealOrangeOne**
A page will be created with this months release on on Tuesday, once the changelog / `pi-image` build as been finalised.

### CODE REVIEW

**Comment by PeterJCLaw** [comment]
@RealOrangeOne why does this PR include a change to the theme?

**Comment by RealOrangeOne** [comment]
@PeterJCLaw I incremented to test something, but apparently committed it accidentally. Have reverted

File: config.yml [hunk 1]
```diff
@@ -36,3 +36,5 @@ outputs:
   home:
     - html
     - json
+
+buildFuture: true
```
Comment (PeterJCLaw) [inline config.yml]: Not sure why this is included in this PR, though I think it's fairly harmless at the moment.

Comment (RealOrangeOne) [reply config.yml]: Its so we can set the real date of an event in the future, and it be rendered in testing / preview

**Pull request merged**